rpc/src/v1/types/trace.rs

```diff
@@ -240,8 +244,6 @@ impl From<trace::Create> for Create {
 #[derive(Debug, Serialize)]
 #[serde(rename_all = "lowercase")]
 pub enum CallType {
-	/// None
-	None,
```

Comment:
Since all the trouble is coming from introducing this variant here, would it maybe be better to leave the None type on the enum directly? Does it make much of a difference to have an enum with 5 variants vs a nested enum (Option where some has 4 variants)?

Reply:
The difference is that `Option<CallType>` and `CallType` from 2.6 have different rlp serialization layout, the former is a list, and the latter is serialized as an int. If we want to be backwards compatible with both 2.6 and 2.7.0, we need to be able to deserialize both versions.